src/modules/logicModule/LM_PC_FundingPot_v1.sol

```diff
@@ -571,27 +578,27 @@ contract LM_PC_FundingPot_v1 is
     /// @notice Validates cap constraints and adjusts amount if needed
     /// @param roundId_ ID of the round
     /// @param amount_ Requested contribution amount
-    /// @param round Round storage object
-    /// @param privileges Access criteria privileges
-    /// @param canOverrideTimeAndCap Whether the user can override cap constraints
+    /// @param round_ Round storage object
+    /// @param privileges_ Access criteria privileges
+    /// @param canOverrideTimeAndCap_ Whether the user can override cap constraints
     /// @return adjustedAmount The potentially adjusted contribution amount
     function _validateAndAdjustCaps(
```

Comment:
We should be exposing this functions in the exposed contract and testing them internally. This is a great way to test these functions directly and make sure there's no bugs especially once we turn them to fuzz tests, my guess is with all this logics there's probably some edge cases in here.